### PR TITLE
chore(replit): use correct brand name across codebase

### LINKED ISSUE #41444: Repl Migration
Just putting this here to keep on radar:

Replit are moving from `repl.it` to `replit.com`.
- https://blog.repl.it/dotcom

They say:
> Nothing is required on your end -- all urls will automatically redirect.

---

The whole platform needs to be audited for the change from `repl.it` to `Replit`, as per:
- https://github.com/freeCodeCamp/freeCodeCamp/pull/41933/files#r621498674

### PR DESCRIPTION
**Pull request by GemmaAprendiendo**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #41444

I hope this PR is OK.  I forgot to refresh before I changed so got a ton of conflicts and was a bit confused on resolving them, but I think I got it right.  These changes are only for the English curriculum.  Chinese and Spanish still have the repl.it.

### CODE REVIEW

File: curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/personal-library.md
Comment (ShaunSHamilton) [inline curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/personal-library.md:14]:
```suggestion
-   Use [our Replit starter project](https://replit.com/github/freeCodeCamp/boilerplate-project-library) to complete your project.
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Thanks, for getting this out.

One minor thing that caught my eye - probably was included a few PRs ago.

**Comment by ShaunSHamilton** [comment]
Also, you likely had issues, because you are working off of your main branch. This is discouraged, as it soils your main after a merge, and prevents you from easily working on other PRs until this is merged.

Here is the recommended workflow: https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally?id=making-changes-locally

**Comment by raisedadead** [comment]
> I forgot to refresh before I changed so got a ton of conflicts and was a bit confused on resolving them, but I think I got it right.

We recommend using branches to avoid frustrations like this. No worries, I think you are almost in the clear. However please give the contributing docs a good read.

**Comment by gmfuster** [comment]
@raisedadead  @ShaunSHamilton  Yes I think I only have one branch.  I'll check that out again and try with 2 branches.  I am using GitHub desktop though, I'm guessing that does not matter.  I'll make the additional change about the parenthesis tomorrow.  I think I'll do that one before I try to get 2 branches, just in case I mess it up.  But I'll refresh right before.  Again, thank you for your help.

**Comment by raisedadead** [comment]
@gmfuster do not worry about this PR. We have added the requested changes. @ShaunSHamilton will review this when he gets a chance. 

But this looks mostly good.

Here is what you should do next:

1. Wait for this PR to be merged.
2. Fix your fork and your local copy on the machine by following these instructions: https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally?id=making-changes-locally

You will be needing to this **after** this PR has been merged and **before** you send in your next PR.

**Review by raisedadead — APPROVED** [review]

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍 . Thank you, for your contribution. We look forward to more just like it!

**Pull request merged**
